api/src/main/resources/messages.properties

```diff
@@ -2711,6 +2711,8 @@ Visit.error.startDate.required=Start date is required
 Visit.failedToAddEncounter=Failed to add the encounter
 Visit.failedToRemoveEncounter=Failed to remove the encounter
 Visit.error.endDateBeforeStartDate=Stop date cannot be before start date
+Visit.error.startDateBeforeBirthDate=Start date cannot be before patient's date of birth
+Visit.error.birthDateTooLess=Patient must be at least 1 year old
```

Comment:
@dkayiwa 

I had a doubt on this one actually....But it was confirmed when the issue mentioned to add patients to be at least one year old.....If that is not the case please do let me know....I will modify the code

Thank You :)